install-dev/classes/controllerHttp.php

```diff
@@ -118,6 +118,11 @@ private function initSteps()
                 'displayName' => $this->translator->trans('Store information', [], 'Install'),
                 'controllerClass' => 'InstallControllerHttpConfigure',
             ],
+            [
+                'name' => 'content',
+                'displayName' => $this->translator->trans('Content information', [], 'Install'),
```

Comment:
Could you tell me where this string appears, please?

Reply:
On the left menu & title of the page